tests/test_parsers.py

```diff
@@ -245,3 +251,70 @@ def test_parse_obographs_merged(self):
             )
         msdf = parse_sssom_table(outfile)
         self.assertTrue(custom_curie_map.items() <= msdf.prefix_map.items())
+
+
+class TestParseExplicit(unittest.TestCase):
+    """This test case contains explicit tests for parsing."""
+
+    def test_round_trip(self):
+        """Explicitly test round tripping."""
+        rows = [
+            (
+                "DOID:0050601",
+                "ADULT syndrome",
+                "skos:exactMatch",
+                "UMLS:C1863204",
+                "ADULT SYNDROME",
+                "semapv:ManualMappingCuration",
+                "orcid:0000-0003-4423-4370",
+            )
+        ]
+        columns = [
+            "subject_id",
+            "subject_label",
+            "predicate_id",
+            "object_id",
+            "object_label",
+            "mapping_justification",
+            "creator_id",
+        ]
+        df = pd.DataFrame(rows, columns=columns)
+        msdf = MappingSetDataFrame(df=df, converter=ensure_converter())
+        msdf.clean_prefix_map(strict=True)
+        self.assertEqual(
+            {"DOID", "semapv", "orcid", "skos", "UMLS"}.union(SSSOM_BUILT_IN_PREFIXES),
```

Comment:
```suggestion
            {"DOID", "orcid", "UMLS"}.union(SSSOM_BUILT_IN_PREFIXES),
```

Reply:
(semapv and skos are built in prefixes)

Reply:
Yes, I have put them here because they are explicitly included. Ultimately, this makes the same set, but I can remove them if it's confusing

Reply:
If it makes sense to you, its fine, I thought i was just and oversight!

Reply:
Addressed in 4d91007